src/test/java/de/tum/in/www1/artemis/exam/StudentExamIntegrationTest.java

```diff
@@ -2780,6 +2801,62 @@ void testConductionOfTestExam_successful() throws Exception {
         }
     }
 
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
+    void testAbandonStudentExam() throws Exception {
+        request.postWithoutLocation("/api/courses/" + course1.getId() + "/exams/" + exam1.getId() + "/student-exams/abandon", studentExam1, HttpStatus.OK, null);
+        StudentExam abandonedStudentExam = studentExamRepository.findById(studentExam1.getId()).orElseThrow();
+        // Ensure that student exam has been marked as abandoned
+        assertThat(abandonedStudentExam.isAbandoned()).isTrue();
+    }
+
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
+    void testAbandonStudentExamAlreadyAbandoned() throws Exception {
+        studentExam1.setAbandoned(true);
+        studentExamRepository.save(studentExam1);
+        request.postWithoutLocation("/api/courses/" + course1.getId() + "/exams/" + exam1.getId() + "/student-exams/abandon", studentExam1, HttpStatus.OK, null);
+        studentExam1 = studentExamRepository.findById(studentExam1.getId()).orElseThrow();
+        // Ensure that student exam is still abandoned
+        assertThat(studentExam1.isAbandoned()).isTrue();
+    }
+
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
+    void testAbandonStudentExamAlreadySubmitted() throws Exception {
+        studentExam1.setSubmitted(true);
+        studentExamRepository.save(studentExam1);
+        request.post("/api/courses/" + course1.getId() + "/exams/" + exam1.getId() + "/student-exams/abandon", studentExam1, HttpStatus.BAD_REQUEST);
+        studentExam1 = studentExamRepository.findById(studentExam1.getId()).orElseThrow();
+        // Ensure that student exam is still submitted and not abandoned
+        assertThat(studentExam1.isSubmitted()).isTrue();
+        assertThat(studentExam1.isAbandoned()).isFalse();
+    }
+
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
+    void testAbandonStudentExamNotInTime() throws Exception {
+        // Forbidden because user tried to abandon before start
+        exam1.setStartDate(ZonedDateTime.now().plusHours(1));
+        examRepository.save(exam1);
+        request.post("/api/courses/" + course1.getId() + "/exams/" + exam1.getId() + "/student-exams/abandon", studentExam1, HttpStatus.FORBIDDEN);
+        // Forbidden because user tried to abandon after end
+        exam1.setStartDate(ZonedDateTime.now().minusHours(5));
+        examRepository.save(exam1);
+        request.post("/api/courses/" + course1.getId() + "/exams/" + exam1.getId() + "/student-exams/abandon", studentExam1, HttpStatus.FORBIDDEN);
```

Comment:
Since the control flow of the respective method in the StudentExamRessource is very simple, we can be certain that the studentExam has not changed when we receive the HttpStatus `forbidden` and it is not necessary to include an additional check here.